src/Controls/tests/DeviceTests/Controls.DeviceTests.csproj

```diff
@@ -19,6 +19,9 @@
 
   <ItemGroup>
     <MauiImage Include="Resources\Images\*" />
+    <MauiImage Remove="Resources\Images\red-embedded.png" />
+    <None Remove="Resources\Images\red-embedded.png" />
```

Comment:
DO we need the none here @mattleibow  ?

Reply:
not sure. probably not but it was in the OG PR so probably no harm?

Reply:
Ah, actually, we need to remove from the MauiImage and None as those do things. We _just_ want the file in EmbeddedResource

Reply:
ok let's remove from above ?